src/components/02-components/cta-card/cta-card.tsx

```diff
@@ -0,0 +1,74 @@
+import { Button, Paper, Typography } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+
+export type BrandProps = {
+  heading: string | null;
+  subHeading: string | null;
+  description: string | null;
```

Comment:
This works, but if the prop optional is better this way:

```suggestion
  heading?: string;
  subHeading?: string;
  description?: string;
```